benchmarking/report-storage.json

```diff
@@ -0,0 +1,146 @@
+{
```

Comment:
report files should not be versioned, can you please add them in git ignore ?
For CI we can publish them as artifacts